pgx-macros/src/lib.rs

```diff
@@ -290,12 +290,12 @@ extension_sql!(r#"\
 );
 
 #[pg_extern(immutable)]
-fn complex_in(input: &std::ffi::CStr) -> PgBox<Complex> {
+fn complex_in(input: &pgx::cstr_core::CStr) -> PgBox<Complex> {
```

Comment:
Hmm it'd be nice if we could provide like a `from` impl for this in `std` code so that users could just `.into()` it.